examples/helm/kanister/kanister-elasticsearch/values.yaml

```diff
@@ -107,3 +313,14 @@ profile:
 # Configure Kanister - the blueprint CR is installed in the controller namespace
 kanister:
   controller_namespace: kasten-io
+  
+## Sysctl init container to setup vm.max_map_count
+# see https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html
+# and https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-configuration-memory.html#mlockall
+sysctlInitContainer:
+  enabled: true
+## Chown init container to change ownership of data and logs directories to elasticsearch user
+chownInitContainer:
+  enabled: true
+## Additional init containers
+extraInitContainers: |
```

Comment:
It looks like you missed a few lines here: 
https://github.com/helm/charts/blob/master/stable/elasticsearch/values.yaml#L317